wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16470_IMU.java

```diff
@@ -1136,6 +1136,18 @@ public synchronized double getAngle(IMUAxis axis) {
     return 0.0;
   }
 
+  /**
+   * Returns the Z axis angle in degrees (CCW positive).
+   *
+   * @return The Z axis angle in degrees (CCW positive).
+   */
+  public synchronized double getAngle() {
+    if (m_simGyroAngleZ != null) {
```

Comment:
The behaviour of these methods in previous years was to read the yaw axis. Adding back in a no-parameter overload should behave the same.

Reply:
Based off #6180 I thought getting the value from the Z axis would do this?

Reply:
all good it should be fixed now